apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -1003,6 +1003,14 @@ public function getInheritedShares(string $path): DataResponse {
 		return new DataResponse(array_values($shares));
 	}
 
+	/**
+	 * Check wether a permissions set contains some permissions.
+	 * @return bool
+	 */
+	private function hasPermission(int $permissionsSet, int $permissionsToCheck) {
```

Comment:
```suggestion
	private function hasPermission(int $permissionsSet, int $permissionsToCheck): bool {
```
Inline return type instead of `@return` comment?